diff --git a/dist/util/resolve.js b/dist/util/resolve.js
index 68625206d92e705ad6a16bf6e2c9fc43881cfb94..13d6fbc570d0b4897270168b2330aa10fd8b1559 100644
--- a/dist/util/resolve.js
+++ b/dist/util/resolve.js
@@ -13,7 +13,7 @@ const createSyncModuleResolver = (extensions, tsConfigFile) => {
     const baseOptions = {
         extensions,
         extensionAlias,
-        conditionNames: ['require', 'import', 'node', 'default'],
+        conditionNames: ['monorepo', 'require', 'import', 'node', 'default'],
         nodePath: false,
     };
     const resolver = new ResolverFactory({
@@ -42,7 +42,7 @@ export const _createSyncModuleResolver = (extensions, tsConfigFile) => timerify(
 const createSyncResolver = (extensions) => {
     const resolver = new ResolverFactory({
         extensions,
-        conditionNames: ['require', 'import', 'node', 'default'],
+        conditionNames: ['monorepo', 'require', 'import', 'node', 'default'],
         nodePath: false,
     });
     resolverInstances.push(resolver);
